src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -1099,3 +1108,459 @@ end
+        p::FT
+    ) where {FT}
+
+    Computes the viscosity of water in Pa s given temperature T (K) and pressure p (Pa)
+    according to Huber et al. (2009) [https://doi.org/10.1063/1.3088050]. 
+
+    Can consider simplifying if this level of precision is not needed
+"""
+function viscosity_h2o(T::FT, p::FT) where {FT}
+    # Reference constants
+    tk_ast  = FT(647.096)    # K
+    ρ_ast = FT(322.0)      # kg/m^3
+    μ_ast  = FT(1e-6)       # Pa s
+
+    # Get density of water [kg/m^3]
+    ρ = density_h2o(T, p)
+
+    # Dimensionless variables
+    tbar  = T / tk_ast
+    tbarx = sqrt(tbar)
+    tbar2 = tbar^2
+    tbar3 = tbar^3
+    ρbar  = ρ / ρ_ast
+    
+    # Calculate μ0 (Eq. 11 & Table 2)
+    μ0 = FT(1.67752) + FT(2.20462) / tbar + FT(0.6366564) / tbar2 - FT(0.241605) / tbar3
+    μ0 = FT(1e2) * tbarx / μ0
+
+    # Coefficients h_array from Table 3
+    h_array = FT.([
+        0.520094    0.0850895   -1.08374   -0.289555   0.0         0.0;
+        0.222531    0.999115     1.88797    1.26613    0.0         0.120573;
+       -0.281378   -0.906851    -0.772479  -0.489837  -0.257040    0.0;
+        0.161913    0.257399     0.0        0.0        0.0         0.0;
+       -0.0325372   0.0          0.0        0.0698452  0.0         0.0;
+        0.0         0.0          0.0        0.0        0.00872102  0.0;
+        0.0         0.0          0.0       -0.00435673 0.0        -0.000593264
+    ])
+
+    # Compute μ1 (Eq. 12 & Table 3)
+    μ1 = FT(0.0)
+    ctbar = (FT(1.0) / tbar) - FT(1.0)
+    for i in 1:6
+        coef1 = ctbar^(i - 1)
+        coef2 = FT(0.0)
+        for j in 1:7
+            coef2 += h_array[j, i] * (ρbar - FT(1.0))^(j - 1)
+        end
+        μ1 += coef1 * coef2
+    end
+    μ1 = exp(ρbar * μ1)
+
+    μ_bar = μ0 * μ1       # Eq. 2
+    μ = μ_bar * μ_ast     # Eq. 1
+    return FT(μ)          # Pa s
+end
+
+
+"""
+    compute_viscosity_ratio(
+        T::FT, 
+        p::FT
+    ) where {FT}
+
+    Computes η*, the ratio of the viscosity of water at temperature T and pressure p
+    to the viscosity of water at STP. 
+"""
+function compute_viscosity_ratio(T::FT, p::FT) where {FT} 
+    η25 = viscosity_h2o(FT(298.15), FT(101325.0))
+    ηstar = viscosity_h2o(T, p) / η25
+    return FT(ηstar)
+end
+
+
+"""
+    po2(P_air::FT) 
+
+    Computes the partial pressure of O2 in the air (Pa) given atmospheric pressure. Assumes a 
+    constant mixing ratio of 21000 / 101325. 
+"""
+function po2(P_air::FT) where {FT} 
+    return FT(0.2095) * P_air 
+end
+
+"""
+    co2_compensation_p(
+        T::FT,
+        To::FT,
+        p::FT,
+        R::FT, 
+        ΔHΓstar::FT = FT(37830),
+        Γstar25::FT = FT(4.332) 
+    ) where {FT}
+
+    Computes the CO2 compensation point (`Γstar`), in units Pa, as a function of temperature T (K)
+    and pressure p (Pa). See Equation B5 of Stocker et al. (2020). 
+"""
+function co2_compensation_p(
+    T::FT,
+    To::FT,
+    p::FT,
+    R::FT, 
+    ΔHΓstar::FT = FT(37830),
```

Comment:
Likewise with these - Let's not supply defaults for these, and make them regular arguments?